addons/mrp/models/mrp_workorder.py

```diff
@@ -159,6 +160,19 @@ def _onchange_finished_lot_id(self):
             if line:
                 self.qty_producing = line.qty_done
 
+    @api.onchange('date_planned_start', 'date_planned_finished')
+    def _onchange_planned_dates(self):
+        if self.date_planned_start and self.date_planned_finished:
+            diff = fields.Datetime.from_string(self.date_planned_finished) - fields.Datetime.from_string(self.date_planned_start)
```

Comment:
Is it usefull to call `fields.Datetime.from_string` as `date_planned_start` and `date_planned_finished` are already Datetime ?